data/data-pipeline/data_pipeline/etl/sources/census_acs/etl.py

```diff
@@ -114,6 +114,27 @@ def __init__(self):
         )
         self.HIGH_SCHOOL_ED_FIELD = "Percent individuals age 25 or over with less than high school degree"
 
+        # College attendance fields
+        self.COLLEGE_ATTENDANCE_TOTAL_POPULATION_ASKED = (
+            "B14004_001E"  # Estimate!!Total
+        )
+        self.COLLEGE_ATTENDANCE_MALE_ENROLLED_PUBLIC = "B14004_003E"  # Estimate!!Total!!Male!!Enrolled in public college or graduate school
+        self.COLLEGE_ATTENDANCE_MALE_ENROLLED_PRIVATE = "B14004_008E"  # Estimate!!Total!!Male!!Enrolled in private college or graduate school
+        self.COLLEGE_ATTENDANCE_FEMALE_ENROLLED_PUBLIC = "B14004_019E"  # Estimate!!Total!!Female!!Enrolled in public college or graduate school
+        self.COLLEGE_ATTENDANCE_FEMALE_ENROLLED_PRIVATE = "B14004_024E"  # Estimate!!Total!!Female!!Enrolled in private college or graduate school
+
+        self.COLLEGE_ATTENDANCE_FIELDS = [
+            self.COLLEGE_ATTENDANCE_TOTAL_POPULATION_ASKED,
+            self.COLLEGE_ATTENDANCE_MALE_ENROLLED_PUBLIC,
+            self.COLLEGE_ATTENDANCE_MALE_ENROLLED_PRIVATE,
+            self.COLLEGE_ATTENDANCE_FEMALE_ENROLLED_PUBLIC,
+            self.COLLEGE_ATTENDANCE_FEMALE_ENROLLED_PRIVATE,
+        ]
+
+        self.COLLEGE_ATTENDANCE_FIELD = (
+            "Percent enrollment in college or graduate school"
+        )
+
         self.RE_FIELDS = [
```

Comment:
What does RE stand for?

Reply:
That's from prior work, but it's race/ethniciy.

Reply:
Thanks